Tests/GeoLib/TestLineSegmentIntersect2d.cpp

```diff
@@ -50,18 +50,29 @@ class LineSegmentIntersect2dTest : public testing::Test
 	ac::gtest_reporter gtest_reporter;
 };
 
+#if not defined(_MSC_VER) && _MSC_VER < 1900
+// Compilers of MVS below 2015 do not support unrestricted unions. The
+// unrestricted union is used by autocheck to handle test data. The autocheck
+// workaround for MVS compilers (below version 2015) contains a bug and in the
+// consequence the tests crashes. For this reason the tests are disabled under
+// this environments.
```

Comment:
- On MSVC++ the tests can be runned only with ` _MSC_VER >= 1900` that supports unrestricted unions.
- I hoped I explained the reason for disabling the tests in the comment below the `#if ...`. What can be improved in the explanation?
- What is the advantage to "#if ... #else ... #endif" each particular test instead of all tests at once?